client/post-editor/editor-drawer/index.jsx

```diff
@@ -12,7 +12,7 @@ import Accordion from 'components/accordion';
 import AccordionSection from 'components/accordion/section';
 import Gridicon from 'components/gridicon';
 import CategoriesTagsAccordion from 'post-editor/editor-categories-tags/accordion';
-import EditorSharingAccordion from 'post-editor/editor-sharing/accordion';
+import EditorSharingAccordion from 'async-component!post-editor/editor-sharing/accordion';
```

Comment:
> So, to be clear, for this instance we'd use <AsyncLoad /> directly instead of the loader in this pull request, and defer the loader to a future one?

Sorry, yes. I see this PR as being for usage of `AsyncLoad` and `asyncRequire?`, along with the babel plugin required to transform those. Then the `import` `webpack` loader being in another PR, so we're not changing too much at once :)